### PR TITLE
Fix db_demo.yaml to use instruments instead of telescopes in photometry and add P48 telescope and ZTF instrument

### PR DESCRIPTION
**Pull request by scizen9**
…hotometry point to ZTF instrument

### CODE REVIEW

**Comment by scizen9** [comment]
I could not find precise values for latitude, longitude and elevation for P48.  Those values are from the [Wikepedia article](https://en.wikipedia.org/wiki/Samuel_Oschin_telescope) on the P48 and may not be precise.

File: data/db_demo.yaml [hunk 1]
```diff
@@ -273,6 +294,20 @@ sources:
     group_ids:
       - =public_group_id
       - =program_A
+  - id: ZTFe028h94k
```
Comment (dannygoldstein) [inline data/db_demo.yaml:297]: are these real names ?

**Review by acrellin — APPROVED** [review]

File: data/db_demo.yaml (hunk 1 above)
Comment (scizen9) [reply data/db_demo.yaml:297]: Yes.  They are the names on the ZTF variable marshal.

Comment (dmitryduev) [reply data/db_demo.yaml:297]: (true, but they are auto-generated and random)

**Pull request merged**